lucet-wasi/src/runtime.rs

```diff
@@ -25,354 +12,3 @@ pub mod types {
 pub fn export_wasi_funcs() {
     hostcalls::init()
 }
-
-pub struct LucetWasiCtx<'a> {
-    vmctx: &'a Vmctx,
-}
-
-impl<'a> LucetWasiCtx<'a> {
-    pub fn wasi(&self) -> Ref<WasiCtx> {
-        self.vmctx.get_embed_ctx()
-    }
-}
-
-impl<'a> types::GuestErrorConversion for LucetWasiCtx<'a> {
-    fn into_errno(&self, e: GuestError) -> types::Errno {
-        debug!("Guest error: {:?}", e);
-        e.into()
-    }
-}
-
-impl<'a> types::UserErrorConversion for LucetWasiCtx<'a> {
-    fn errno_from_error(&self, e: Error) -> Result<types::Errno, lucet_wiggle::Trap> {
-        debug!("Error: {:?}", e);
-        e.try_into()
-            .map_err(|e| lucet_wiggle::Trap::String(format!("{:?}", e)))
-    }
-}
-
-impl<'a> wasi_snapshot_preview1::WasiSnapshotPreview1 for LucetWasiCtx<'a> {
```

Comment:
Have these moved somewhere else, or are they somehow obviated by the new design?

Reply:
These are obviated - the redesign of lucet-wiggle for using `WasiCtx` directly as the ctx type, rather than the lucet-specific wrapper, means we can use the impls from wasi-common rather than delegate to them with this boilerplate.